style.css

```diff
@@ -0,0 +1,12 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
```

Comment:
According to the project requirement, you're supposed to add one class with styles for your header in `index.html`
Kindly add the styling.